src/applications/check-in/pages/CheckIn.jsx

```diff
@@ -28,36 +52,44 @@ const CheckIn = props => {
           className="appointment-details vads-u-font-weight--bold vads-u-font-family--serif"
           data-testid="appointment-date"
         >
-          Friday, September 25, 2020
+          {appointmentDate}
         </dd>
         <dd
           className="appointment-details vads-u-font-weight--bold vads-u-margin-bottom--3 vads-u-font-family--serif"
           data-testid="appointment-time"
         >
-          9:30 a.m. ET
+          {appointmentTime} {timeZone}
         </dd>
         <dt className="vads-u-font-weight--bold vads-u-margin--0 vads-u-margin-right--1">
           Clinic:{' '}
         </dt>
-        <dd data-testid="clinic-name">Green Team Clinic1</dd>
+        <dd data-testid="clinic-name">{appointment.clinicName}</dd>
       </dl>
       <button
         type="button"
         className="usa-button usa-button-big"
         onClick={onClick}
         data-testid="check-in-button"
+        disabled={isLoading}
       >
-        Check in now
+        {isLoading ? <>Loading...</> : <>Check in now</>}
```

Comment:
not following the situation where a the app would have rendered (that is, past the default va.gov spinner), but the actual check-in button would somehow still be loading? not a big deal, just want to make sure im understanding.

Reply:
Great question! That is after the user has clicked the button and the app is sending the API call to check the user in. That call will probably take a little bit of time. So that is just the immediate feedback for the user. The actual text itself is still being finalized.

Reply:
ahhhh got it. thank you!